subsys/bluetooth/controller/ll_sw/ull_slave.c

```diff
@@ -230,8 +230,8 @@ void ull_slave_setup(memq_link_t *link, struct node_rx_hdr *rx,
 		rx = adv->lll.node_rx_adv_term;
 		link = rx->link;
 
-		rx->handle = ull_adv_handle_get(adv);
 		rx->type = NODE_RX_TYPE_EXT_ADV_TERMINATE;
+		rx->handle = ull_adv_handle_get(adv);
```

Comment:
Seems like an unnecessary change.

Reply:
This is make it consistent, to set the handle after the type.